docs/Displaying a User Interface/Popup Keyboards/index.md

```diff
@@ -202,14 +202,66 @@ KeyboardListener keyboardListener = new KeyboardListener() {
 ```
 !@
 
-@![iOS, android, javaSE, javaEE]
+@![javascript]
+```javascript
+const keyboardListener = new KeyboardListener()
+    .setOnUserDidSubmitInput((event) => {
```

Comment:
setOnUserDidSubmitInput's function input has an `inputText` argument first, the `event` argument second